plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1526,9 +1526,9 @@ fc::variant read_only::get_block_header_state(const get_block_header_state_param
    return vo;
 }
 
-void read_write::push_block(const read_write::push_block_params& params, next_function<read_write::push_block_results> next) {
+void read_write::push_block(read_write::push_block_params params, next_function<read_write::push_block_results> next) {
```

Comment:
`read_write::push_block_params params` could be `read_write::push_block_params&& params` to avoid a copy.